pages/05_embedding_in_other_apps.md

```diff
@@ -0,0 +1,35 @@
+# Embedding in Sharepoint, Salesforce, and Mobile Apps
+
+Much of this playbook focuses on embedding Tableau content in custom-developed web applications, but there are three common embedding scenarios that have specific requirements: Sharepoint, Salesforce, and Mobile Apps.
+
+## Embedding in Sharepoint
+Tableau provides a SharePoint Web Part to facilitate SharePoint embedding scenarios. 
+
+The Web Part, as well as the instructions for installing it are located under `C:\Program Files\Tableau\Tableau Server\<version>\extras\embedding\sharepoint` within your Tableau Server install directory.
```

Comment:
The sample code is for trusted tickets which are for on-prem, not Online. Added a comment that you can embed Online views into Sharepoint but AD and tickets don't work.